src/modules/wkdLocate.js

```diff
@@ -61,24 +61,61 @@ export async function lookup(email) {
     return;
   }
 
-  const url = await buildWKDUrl(email);
+  let data;
 
-  // Impose a size limit and timeout similar to that of gnupg.
-  const data = await timeout(TIMEOUT * 1000, window.fetch(url)).then(
-    res => sizeLimitResponse(res, SIZE_LIMIT * 1024));
+  /** For the WKD standard (draft version 13, https://datatracker.ietf.org/doc/draft-koch-openpgp-webkey-service/13/) it is important to check, if the subdomain openpgpkey exists. Only in that
+  * case we should use the advanced method. The optimal way to do this check, is to try to
+  * resolve the DNS name. On the 21st of November 2021 only Firefox (since version 60) supported the method dns.resolve()
+  * (https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/dns/resolve).
+  * That's why we check first, if this method exists.
+  */
+  if (typeof browser !== 'undefined' && typeof(browser.dns.resolve) === 'function') {
```

Comment:
Check on existence of browser not required as we use webextension-polyfill

Reply:
I've changed that part